<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>149</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>15777215</width>
    <height>15777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="stat_display_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>google</string>
    </property>
   </widget>
   <widget class="QWidget" name="kakao_led_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="naver_led_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>naver</string>
    </property>
   </widget>
   <widget class="QWidget" name="google_led_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kakao</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
